gulpfile.js

```diff
@@ -107,88 +107,96 @@ var browserifyOptions = {
 };
 
 var ugliyOptions = {
-    compress:{
-        dead_code     : true,  // jshint ignore:line
-        drop_debugger : true,  // jshint ignore:line
-        global_defs   : {      // jshint ignore:line
+    compress: {
+        dead_code: true,  // jshint ignore:line
+        drop_debugger: true,  // jshint ignore:line
+        global_defs: {      // jshint ignore:line
             "DEBUG": false      // matters for some libraries
         }
     }
 };
 
-gulp.task('version', function(){
-  if(!version.version) return;
-
-  gulp.src(['./package.json'])
-    .pipe(replace(/\"version\"\: \"([\.0-9\-a-z]*)\"/, '"version": "'+ version.version + '"'))
-    .pipe(gulp.dest('./'));
-  gulp.src(['./bower.json'])
-    .pipe(replace(/\"version\"\: \"([\.0-9\-a-z]*)\"/, '"version": "'+ version.version + '"'))
-    .pipe(gulp.dest('./'));
-  gulp.src(['./package.js'])
-    .pipe(replace(/version\: \'([\.0-9\-a-z]*)\'/, "version: '"+ version.version + "'"))
-    .pipe(gulp.dest('./'));
+gulp.task('version', function () {
+    if (!lernaJSON.version) {
+        throw new Error("version property is missing from lerna.json");
+    }
+
+    var version = lernaJSON.version;
+    var jsonPattern = /\"version\"\: \"[\.0-9\-a-z]*\"/;
+    var jsPattern = /version\: \'[\.0-9\-a-z]*\'/;
+    var meteorPattern = /ethereum:web3@[\.0-9\-a-z]*/;
```

Comment:
This we also can remove, as we can’t modify the versions Json, this will be done by meteor publish.

Currently this is a manual thing,as meteor doesn’t allow to exclude certain folders, I have to manually copy them out to publish a release.

Reply:
OK, removed.